### PR TITLE
add Minimum Number of Operations to Move All Balls to Each Box leetcode

### PR DESCRIPTION
**Pull request by alexpantyukhin**
#### Description of Change

Add a guide for new LeetCode solutions.

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: leetcode/src/1769.c [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
+
```
Comment (Panquesito7) [inline leetcode/src/1769.c:2]:
```suggestion
```

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🚀

File: leetcode/src/1769.c [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/**
+ * Note: The returned array must be malloced, assume caller calls free().
+ */
+
+// Count one's from right. Each step from right side decrease for one for each 1's and increase from left:
+// 1001*0101  -> left: 4 + 1, right: 2 + 4
+// 10010*101  -> left: (4+1) + (1+1), right: (2-1) + (4-1)
+// Runtime: O(n)
+// Space: O(1)
+int* minOperations(char * boxes, int* returnSize){
```
Comment (tjgurwara99) [inline leetcode/src/1769.c:10]: Could you make this more consistent?

Either 
```suggestion
int* minOperations(char* boxes, int* returnSize){
```
which is what I prefer or
```suggestion
int * minOperations(char * boxes, int * returnSize){

```

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]

**Comment by alexpantyukhin** [comment]
review notes are fixed

**Comment by alexpantyukhin** [comment]
review notes are fixed

**Review by tjgurwara99 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**